lib/plugins/inspec-init/lib/inspec-init/renderer.rb

```diff
@@ -28,7 +28,10 @@ def render_with_values(template_type, template_values = {})
         template_values[:name] = template_values[:name].split(%r{\\|\/}).last
         # Generate the full full_destination_root_path path on disk
         full_destination_root_path = Pathname.new(Dir.pwd).join(profile_path)
-        ui.plain_text "Create new #{template_type} at #{ui.mark_text(full_destination_root_path)}"
+
+        # This is a bit gross
+        generator_type = template_subdir_path.split(%r{[\/]}).first.sub(/s$/, '')
```

Comment:
We may need to do a `Pathname.new(template_subdir_path).parent`(or something like that) here. I am worried about slashes on windows.

Reply:
I guess you do a File.join before this so it should covert any accordingly.